public/usage-examples/graphics/fill_triangle/fill_triangle-1-simple-oop.cs

```diff
@@ -0,0 +1,16 @@
+using SplashKitSDK;
+
+namespace Program
+{
+  public class Program
+  {
+    public static void Main()
+    {
+      Window myWindow = SplashKit.OpenWindow("Simple Red Triangle", 800, 600);
+      SplashKit.FillTriangle(SplashKit.ColorRed(), 100, 100, 200, 200, 300, 100);
+      SplashKit.RefreshWindow(myWindow);
+      SplashKit.Delay(3000);
+      SplashKit.CloseWindow(myWindow);
+    }
+  }
+}
```

Comment:
Same here with this one, all you needed to do was take the existing top level code which was - 

```csharp
using SplashKitSDK;
using static SplashKitSDK.SplashKit;

OpenWindow("Fill Triangle Example", 800, 600);
ClearScreen();

FillTriangle(ColorRed(), 100, 100, 200, 200, 300, 100);
RefreshScreen();

Delay(5000);
CloseAllWindows();
```


And then swap using the guide I've linked before. Which would then become - 

```csharp
using SplashKitSDK;

namespace Program
{
    public class Program
    {
        public static void Main()
        {
            SplashKit.OpenWindow("Fill Triangle Example", 800, 600);
            SplashKit.ClearScreen();

            SplashKit.FillTriangle(SplashKit.ColorRed(), 100, 100, 200, 200, 300, 100);
            SplashKit.RefreshScreen();

            SplashKit.Delay(5000);
            SplashKit.CloseAllWindows();
        }
    }
}
```